crates/trie/common/src/proofs.rs

```diff
@@ -34,6 +34,37 @@ impl MultiProofTargets {
         Self(B256Map::with_capacity_and_hasher(capacity, Default::default()))
     }
 
+    /// Create `MultiProofTargets` with a single account as a target.
+    pub fn account(hashed_address: B256) -> Self {
+        Self::accounts([hashed_address])
+    }
+
+    /// Create `MultiProofTargets` with a single account and slots as targets.
+    pub fn account_with_slots<I: IntoIterator<Item = B256>>(
+        hashed_address: B256,
+        slots_iter: I,
+    ) -> Self {
+        Self(B256Map::from_iter([(hashed_address, slots_iter.into_iter().collect())]))
+    }
+
+    /// Create `MultiProofTargets` only from accounts.
+    pub fn accounts<I: IntoIterator<Item = B256>>(iter: I) -> Self {
+        Self(iter.into_iter().map(|hashed_address| (hashed_address, Default::default())).collect())
+    }
```

Comment:
can we add a `new_with_` prefix for these methods?

Reply:
i can do `from_`